spec/view_helper/applinks_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+
+describe MetaTags::ViewHelper, 'displaying Applinks meta tags' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.